src/content/docs/tutorial-errors/scope-the-problem.mdx

```diff
@@ -0,0 +1,159 @@
+
+## Objectives [#objective]
+
+This tutorial series shows you how to solve critical errors, then guides you into reducing your overall error count. This doc covers these concepts:
+
+* Choosing a service to begin error analysis
+* Choosing an error group that indicates an outage 
+
+## Prerequisites [#prereq]
+
+To monitor your application's performance, you will use an agent created specifically for your app's language.
+
+* Click a logo to install an agent. This will send you to the New Relic platform where you will be guided through installing and configuring the agent.
+
+<TechTileGrid>
+  
+  <TechTile
+    name="Go agent"
+    icon="logo-go"
+    to="https://one.newrelic.com/nr1-core?state=985d4005-ba90-a8c7-1da1-2af34539b03b"
+  />
+
+  <TechTile
+    name="Java agent"
+    icon="logo-java"
+    to="https://one.newrelic.com/nr1-core?state=80d18bcb-4919-1fcb-2b77-9406838eb916"
+  />
+
+  <TechTile
+    name=".NET agent"
+    icon="logo-dotnet"
+    to="https://one.newrelic.com/nr1-core?state=30e93090-6dfa-6b70-8e75-472f54414355"
+  />
+
+  <TechTile
+    name="Node.js agent"
+    icon="logo-nodejs"
+    to="https://one.newrelic.com/marketplace/install-data-source?state=be2e62fa-cc3b-c428-27c4-8d662c9e80a1"
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=aa633b41-72d4-009c-3abf-55dcf64894fe"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+
+</TechTileGrid>
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
+
+* [Integrate your logs](/docs/logs/get-started/get-started-log-management/#integrate-logs). The APM agent provides some logging details with logs in context, but certain key attributes could be missing from your log details. We recommend that you invest some time in setting up our full logs solution to expedite the error analysis process.
+
+## Find the critical error [#critical-error]
+
+With your apps instrumented, New Relic can capture data about your services. This includes data about error occurrences in your app. 
+
+<Steps>
+    <Step>
+        ### Think about the end user
+
+Similar to our flood metaphor, it's likely that when an outage occurs, the alerting services only indicate a problem exists somewhere. When you first receive an alert, you won't have enough context to make a decision about where to start in your system. 
```

Comment:
Should you say, "New Relic's alerting services...?"

Reply:
Maybe we should be more positive about alerts? Like-- alerts will let you know that an entity you're monitoring isn't performing as expected. It's like the water all over the lawn. Alerts tell you something bad is happening and where: the lawn. But they don't provide the context. That's where errors come in.

Reply:
Yeah, I like that impulse. I'll tinker around and land on something